apps/storybook-react/.storybook/main.ts

```diff
@@ -47,7 +46,12 @@ const config: StorybookConfig = {
         ),
       };
     }
+
+    // Add asset inclusion for font files so that Vite processes them correctly.
+    config.assetsInclude = ['**/*.woff2', '**/*.woff', '**/*.ttf', '**/*.otf'];
+
```

Comment:
Can we remove this seeing as we aren't fetching the fonts from the design tokens package for the time being?